src/Bridges/Variable/bridges/hermitian.jl

```diff
@@ -332,3 +330,63 @@ function unbridged_map(
 ) where {T}
     return (_variable(bridge, i) => convert(MOI.ScalarAffineFunction{T}, vi),)
 end
+
+function MOI.supports(
+    ::MOI.ModelLike,
+    ::MOI.VariablePrimalStart,
+    ::Type{<:HermitianToSymmetricPSDBridge},
+)
+    return true
+end
+
+# Set the starting value of the diagonal elements of the imaginary part
+function _set_imag_diag_start(
+    model::MOI.ModelLike,
+    attr::MOI.VariablePrimalStart,
+    bridge::HermitianToSymmetricPSDBridge,
+    value,
+)
+    for i in 1:bridge.n
+        k = MOI.Utilities.trimap(i, bridge.n + i)
+        MOI.set(model, attr, bridge.variables[k], value)
+    end
+end
+
+_minus(a) = -a
+_minus(::Nothing) = nothing
+
+function MOI.set(
+    model::MOI.ModelLike,
+    attr::MOI.VariablePrimalStart,
+    bridge::HermitianToSymmetricPSDBridge,
+    value,
```

Comment:
I preferred to have a separate method for the `::Nothing` case. It's a lot cleaner